### PR TITLE
[fix bug 1140456] Create simple video snippet

### PR DESCRIPTION
**Pull request by ckprice**
@glogiotatidis r?

https://bugzilla.mozilla.org/show_bug.cgi?id=1140456

Inspired from
https://github.com/mozilla/snippets/blob/master/templates/10th-anniversary-video/10th-anniversary-video-no-share.html

### CODE REVIEW

File: video-snippets/simple-over-search-bar.html [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    font-weight: normal;
+    font-style: normal;
+}
+
+#simple-video-snippet-{{ snippet_id }} {
+    width: 470px;
+    margin: 0 auto;
+    display: inline-block;
+    text-align: start;
```
Comment (glogiotatidis) [inline video-snippets/simple-over-search-bar.html]: i didn't know that, nice!

File: video-snippets/simple-over-search-bar.html [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+        replay.classList.remove('show');
+    });
+
+    // Track progress through the video.
+    var lastProgress = 0;
+    var thresholds = [0.25, 0.5, 0.75, 1];
+    video.addEventListener('timeupdate', function() {
+        var currentProgress = video.currentTime / video.duration;
+        if (currentProgress > lastProgress && !video.paused) {
```
Comment (glogiotatidis) [inline video-snippets/simple-over-search-bar.html]: So if video has ended, `currentProgress==1` and `video.paused==True` therefore you don't ever send `simple-video-threshold-1`, correct? Is this why you added line 206?

**Comment by glogiotatidis** [comment]
Looks good. Just the metric sending logic is a bit complicated. If we can reftactor this into something simpler it would be great, otherwise we can just go ahead with what we have if it works the way you want.

**Comment by ckprice** [comment]
Thanks for the r @glogiotatidis. Addressed your sendMetric comment. Thoughts?

**Comment by glogiotatidis** [comment]
lgtm, nice work! r+ :cake:

**Pull request merged**